src/wp-includes/post.php

```diff
@@ -328,6 +328,7 @@ function create_initial_post_types() {
 				'title',
 				'editor',
 				'revisions',
+				'custom-fields',
```

Comment:
Is custom fields support actually needed?

Reply:
In my testing the postmeta sync_status field was not returned without this.

Reply:
We are however going to look at returning it via REST as a top-level `sync_status` field rather than as post meta so we may be able to drop custom fields support if we go that way.